CRM/Civicase/Settings.php

```diff
@@ -287,18 +302,38 @@ public static function setCaseCategoryInstanceToJsVars(array &$options): void {
    *   Default values to use when exposing settings.
    */
   public static function exposeSettings(array &$options, array $defaults): void {
-    $options['allowMultipleCaseClients'] = (bool) Civi::settings()->get('civicaseAllowMultipleClients');
-    $options['showComingSoonCaseSummaryBlock'] = (bool) Civi::settings()->get('civicaseShowComingSoonCaseSummaryBlock');
-    $options['allowCaseLocks'] = (bool) Civi::settings()->get('civicaseAllowCaseLocks');
-    $options['allowLinkedCasesTab'] = (bool) Civi::settings()->get('civicaseAllowLinkedCasesTab');
-    $options['showWebformsListSeparately'] = (bool) Civi::settings()->get('civicaseShowWebformsListSeparately');
-    $options['webformsDropdownButtonLabel'] = Civi::settings()->get('civicaseWebformsDropdownButtonLabel');
-    $options['showFullContactNameOnActivityFeed'] = (bool) Civi::settings()->get('showFullContactNameOnActivityFeed');
-    $options['includeActivitiesForInvolvedContact'] = (bool) Civi::settings()->get('includeActivitiesForInvolvedContact');
-    $options['civicaseSingleCaseRolePerType'] = (bool) Civi::settings()->get('civicaseSingleCaseRolePerType');
-    $options['caseTypeCategoriesWhereUserCanAccessActivities'] =
-      CRM_Civicase_Helper_CaseCategory::getWhereUserCanAccessActivities();
-    $options['currentCaseCategory'] = $defaults['caseCategoryId'] ?: NULL;
+    try {
+      $settingsValues = civicrm_api3('Setting', 'get', [
```

Comment:
@shahrukh-compuco  I do not see the explanation in the PR why we need to get this setting from the domain?  That is different between` Civi::settings()->get('civicaseAllowMultipleClients'); `and using` Setting.get A`PI?